lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1267,6 +1269,9 @@ SPIRVValue *LLVMToSPIRV::transValueWithoutDecoration(Value *V,
     return BV ? mapValue(V, BV) : nullptr;
   }
 
+  if (InlineAsm *IA = dyn_cast<InlineAsm>(V))
+    return mapValue(V, transAsmINTEL(IA, BB));
```

Comment:
Please check if corresponding extension is enabled

Reply:
Done